.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,29 @@
+---
+name: 💡 Idea
+about: Describe an idea, improvement, todo, feature, issue, story, or task
+---
+
+# 💡 Summary
+
+What is the work, as a high-level summary?
+
+## Pitch
+
+Why does this feature belong in this project?
+
+This would be useful because...
+
+## Implementation notes
+
+Please provide details for implementation, such as:
+
+- an example for how this feature would be used
+- what this feature would look
+- how this feature would act
+- any related work, especially links to issues
```

Comment:
```suggestion
* an example for how this feature would be used
* what this feature would look
* how this feature would act
* any related work, especially links to issues
```

Reply:
I have a strong preference for dashes over asterisks for building a bulleted list in an issue, but don't have a definable reason other than "the asterisk lists bug my synesthesia."

I defer to the wisdom of the group on what to use.

Reply:
Our linter prefers/requires dashes, so I'll go with that to keep the build from failing.